components/brave_wallet/common/features.cc

```diff
@@ -26,7 +27,12 @@ BASE_FEATURE(kBraveWalletNftPinningFeature,
 
 BASE_FEATURE(kBraveWalletBitcoinFeature,
              "BraveWalletBitcoin",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+#if BUILDFLAG(ENABLE_BITCOIN_BY_DEFAULT)
```

Comment:
features should be enabled/disabled by channel using griffin, not using buildflags

Reply:
seems this isn't even used anymore and should be deleted

Reply:
> seems this isn't even used anymore and should be deleted

ios team will delete this soon, they're almost with btc support afaik.

Reply:
> features should be enabled/disabled by channel using griffin, not using buildflags

- There is no way to enable something in seeds file for dev builds.
- We needed this in nightly and dev builds. Just enabling by default on desktop could get into beta and we didn't want any additional actions to disable it then.
- This approach was already used before https://github.com/brave/brave-core/pull/5898